src/main/webapp/WEB-INF/templates/system/common/navigation.ftl

```diff
@@ -0,0 +1,166 @@
+<#include "/templates/system/common/ice.ftl"/>
+
+<#macro navigation
+<#---->url="/site/website"
```

Comment:
why those empty comments in each line?

Reply:
They are just means of indentation so that the editor code reformat doesn't mess up format.
FTL code formatting is pretty rubbish even in IntelliJ.